src/codeu/chat/client/ClientUser.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.regex.*;
```

Comment:
Generally I'd recommend against wildcard imports, in favor of importing specific classes instead.